migrations/migration_4_standalone_slashing_data.go

```diff
@@ -0,0 +1,69 @@
+package migrations
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/bloxapp/ssv/storage/basedb"
+
+	"go.uber.org/zap"
+)
+
+var migration_4_standalone_slashing_data = Migration{
+	Name: "migration_4_standalone_slashing_data",
+	Run: func(ctx context.Context, logger *zap.Logger, opt Options, key []byte, completed CompletedFunc) error {
+		return opt.Db.Update(func(txn basedb.Txn) error {
+			signerStorage := opt.signerStorage(logger)
+			legacySPStorage := opt.legacySlashingProtectionStorage(logger)
+			spStorage := opt.slashingProtectionStorage(logger)
+
+			accounts, err := signerStorage.ListAccountsTxn(txn)
+			if err != nil {
+				return fmt.Errorf("failed to list accounts: %w", err)
+			}
+
+			for _, account := range accounts {
+				sharePubKey := account.ValidatorPublicKey()
+
+				// migrate highest attestation slashing protection data
+				highAtt, found, err := legacySPStorage.RetrieveHighestAttestation(sharePubKey)
+				if err != nil {
+					return err
+				}
+				if !found {
+					return fmt.Errorf("highest attestation not found for share %s", hex.EncodeToString(sharePubKey))
+				}
+				if highAtt == nil {
+					return fmt.Errorf("highest attestation is nil for share %s", hex.EncodeToString(sharePubKey))
+				}
```

Comment:
@olegshmuelov , @moshe-blox  in case of error, shouldn't we skip `continue` and try next account?

Reply:
No, in this case, an error signifies a critical issue with the slashing data for the specific share. Automatically moving to the next account without addressing the error could potentially lead to incomplete or incorrect data migration, which is why manual intervention is preferred.